cpp/turbodbc_arrow/Library/src/arrow_result_set.cpp

```diff
@@ -81,8 +75,8 @@ namespace {
     epoch.tm_year = 70;
     epoch.tm_mday = 1;
 
-    // Milliseconds since the epoch
-    return lrint(difftime(mktime(&date), mktime(&epoch)) * 1000);
+    // days since the epoch
+    return lrint(difftime(mktime(&date), mktime(&epoch)) / 86400);
```

Comment:
@wesm The master branch already supports MSVC packaging (Python 3.5+) and @xhochy has created turbodbc, so I hope there is not too much to do on the packaging side.